From 6187ca0a33a7b1b372ec5889ddfa02c0f3f83b95 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 2 Mar 2019 20:36:42 +0200
Subject: [PATCH 3/3] Snap: Add back original-package in manifest

* When you upgrade from 14.1 to 15.1 and then 16.0,
  Snap is no longer working because the system is confused about
  the package name.

Change-Id: I5588dbda38063337a01409f19fd6e3b32b4e25ca
---
 AndroidManifest.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 67ae8216b..f4a3180f9 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -7,6 +7,8 @@
         android:minSdkVersion="23"
         android:targetSdkVersion="26" />
 
+    <original-package android:name="org.cyanogenmod.snap" />
+
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
-- 
2.17.1

